reconcile/test/external_resources/test_rds_timeouts.py

```diff
@@ -0,0 +1,39 @@
+from unittest.mock import Mock
+
+import pytest
+
+from reconcile.external_resources.aws import AWSRdsFactory
+from reconcile.external_resources.model import (
+    ExternalResource,
+    ExternalResourceModuleConfiguration,
+    ExternalResourceProvision,
+)
+
+
+def test_timeout_values() -> None:
+    f = AWSRdsFactory(er_inventory=Mock(), secret_reader=Mock())
+
+    assert f._get_timeout_value("60m") == 60
+    assert f._get_timeout_value("1h") == 60
+    with pytest.raises(
+        ValueError,
+        match=r".*Invalid RDS instance timeout format: 12mm. Specify timeout in minutes\(m\) or hours\(h\)..*",
+    ):
+        assert f._get_timeout_value("12mm")
```

Comment:
don't test private method, can make it a public static method or a helper function

Reply:
Changed the test methods to use validate() instead